app/src/main/java/com/eveningoutpost/dexdrip/GcmListenerSvc.java

```diff
@@ -559,6 +560,9 @@ public void run() {
                     }
                 } else if (action.equals("libreBlock")) {
                     HandleLibreBlock(payload);
+                } else if (action.equals("l2rs")) {
+                    Libre2RawValue currentRawValue = Libre2RawValue.fromJSON (payload);
```

Comment:
Please use consistent code formatting (no spaces before bracket of function parameter list), at least for the code you have changed.